README.md

```diff
@@ -107,6 +107,41 @@ NBitcoin is inspired by Bitcoin Core code but provides a simpler object oriented
 
 NBitcoin is licensed under the [MIT License](https://opensource.org/licenses/MIT) and we encourage you to use it to explore, learn, debug, play, share and create software for Bitcoin and with other Metaco services.
 
+## Some OSS projects using NBitcoin
+
+* [Wasabi](http://github.com/zkSNACKs/WalletWasabi): Privacy focused, ZeroLink compliant Bitcoin wallet.
+
+* [StratisBitcoinFullNode](http://github.com/stratisproject/StratisBitcoinFullNode): Bitcoin full node in C# https://stratisplatform.com
+
+* [Breeze](http://github.com/stratisproject/Breeze): Breeze Wallet, the first full-block SPV bitcoin wallet 
+
+* [BlockExplorer](http://github.com/stratisproject/BlockExplorer): A set of projects that can index and query stratis blockchains on the fullnode. 
+
+* [BTCPay](http://github.com/btcpayserver/btcpayserver): A cross platform, self-hosted server compatible with Bitpay API 
+
+* [NTumbleBit](http://github.com/NTumbleBit/NTumbleBit): TumbleBit Implementation in .NET Core 
+
+* [BitPoker](http://github.com/bitcoinbrisbane/BitPoker): Decentralised peer to peer poker, using bitcoin http://www.bitpoker.io
+
+* [Zen-Wallet](http://github.com/zenprotocol/zen-wallet): Node and GUI for the Zen Protocol. https://www.zenprotocol.com
+
+* [Metaco-Trader](http://github.com/MetacoSA/Metaco-Trader): Bitcoin Wallet for advanced user based on a NBitcoin.Server 
+
+* [Swarmops](http://github.com/Swarmops/Swarmops): Admin backend for any bitcoin-native or swarm organization http://sandbox.swarmops.com/
+
+* [Nako](http://github.com/CoinVault/Nako): A Bitcoin and Altcoin server api that indexes blockchain transactions and addresses 
+
+* [NBXplorer](http://github.com/dgarage/NBXplorer): NBitcoin Explorer 
+
+* [NElements](http://github.com/NicolasDorier/NElements): Adapting NBitcoin for NElements 
```

Comment:
Remove this one, outdated fork. I will add element support in NBitcoin.Altcoins soon.